src/Microsoft.Health.Fhir.Shared.Core/Features/Search/BundleFactory.cs

```diff
@@ -71,6 +71,12 @@ public ResourceElement CreateHistoryBundle(SearchResult result)
                 }
 #endif
 
+                if (!hasVerb && string.Equals("Import", r.Resource.Request?.Method, StringComparison.OrdinalIgnoreCase))
+                {
+                    hasVerb = true;
+                    httpVerb = resource.IsDeleted() ? Bundle.HTTPVerb.DELETE : Bundle.HTTPVerb.PUT;
```

Comment:
Should/does the import code do this logic now too?

Reply:
Currently $import saves all its resources with Resource.Method=Import. This is nice as it allows to distinguish records from $import and API.